modules/promo-code/promo-code.permissions.js

```diff
@@ -1,9 +1,14 @@
 const { allow } = require('graphql-shield');
 
+const promoCodeQuery = {
+  getAllPromoCodes: allow,
+  getPromoCodeByCode: allow,
+};
+
 const promoCodeMutation = {
   addPromoCode: allow,
   deletePromoCode: allow,
   updatePromoCode: allow,
 };
```

Comment:
why have all these methods been allowed?
not logged persons can see all promocodes?
please, describe requirements

Reply:
I got it, in Progress

Reply:
Done

Reply:
please, give me requirements for these methods